pkgs/development/ocaml-modules/directories/default.nix

```
@@ -5,7 +5,7 @@ buildDunePackage rec {
   version = "0.5";
   useDune2 = true;
```

```suggestion
  duneVersion = "2";
```
This can also likely be 3